packages/datatrak-web/src/features/Survey/Components/SurveyQuestion.tsx

```diff
@@ -36,21 +37,21 @@ const Placeholder = ({ name, type, id }) => {
 };
 
 export enum QUESTION_TYPES {
+  Autocomplete = Placeholder,
   Binary = BinaryQuestion,
   Checkbox = CheckboxQuestion,
   Date = DateQuestion,
   DateTime = DateTimeQuestion,
   FreeText = TextQuestion,
+  Entity = EntityQuestion,
   Geolocate = GeolocateQuestion,
-  Autocomplete = Placeholder,
   Instruction = InstructionQuestion,
   Number = TextQuestion,
   Photo = Placeholder,
   Radio = RadioQuestion,
   SubmissionDate = DateQuestion,
   DateOfData = DateQuestion,
-  Entity = Placeholder,
-  PrimaryEntity = Placeholder,
+  PrimaryEntity = EntityQuestion,
```

Comment:
There is another ticket for the PrimaryEntity question so this will get updated soon